### PR TITLE
[6] Product Page using HTML 

### PR DESCRIPTION
**Pull request by Lolatech07**
## Task: `Product Page using HTML`
- `Included the product images and added product prices`

Screenshot 1 of 3
Screenshot 2 of 3
Screenshot 3 of 3

### CODE REVIEW

File: submissions/lolaOluwadahunsi/html/Ecommerce website.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            <strong>Email address:</strong>
+            <em>info@laptopsrus.com</em>
+        </p>
+
+
+        <p>
+            <strong>Phone Number:</strong>
+            <em>0801000456</em>
+        </p>
+
```
Comment (adaugochi) [inline submissions/lolaOluwadahunsi/html/Ecommerce website.html:121]: remove spaces from line 121 - 127

File: submissions/lolaOluwadahunsi/html/Ecommerce website.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        <p>
+            <strong>CONTACT:</strong>
+        </p>
+
+        <p>
+            <strong>Email address:</strong>
+            <em>info@laptopsrus.com</em>
+        </p>
+
```
Comment (adaugochi) [inline submissions/lolaOluwadahunsi/html/Ecommerce website.html:115]: Single line space is ok. using a double line space is too much.

**Comment by Lolatech07** [comment]
Thanks. Done.

**Comment by adaugochi** [comment]
Avoid spaces in the name of your file. Both your HTML file and image names has space. Use underscore(_) or hyphen(-) instead of spaces

**Comment by Lolatech07** [comment]
Thanks for your reply. Please are there any other corrections so I can
resolve all at once and not in bits?

**Comment by adaugochi** [comment]
For now, that's what I can see

**Comment by Lolatech07** [comment]
Okay. Thanks.

I have renamed the files.

**Comment by Lolatech07** [comment]
Thanks. HTMl file has been renamed in lower case and the width, height and
content attribute for the images have been removed. The image names have
also been updated.

**Comment by Lolatech07** [comment]
Quick question please...why do I need to remove the width, height and
content attribute for where images are used? I thought these were to
specify the size of the images?

Seems I am missing something.

Kindly help clarify.

Thanks.

**Comment by Lolatech07** [comment]
Thank youuuuuuuuuuu.....!

Kindly grant me access to the next course.

Many thanks for the support.

**Comment by Lolatech07** [comment]
Thank you!!!

Many thanks for the support.

Kindly grant me access to the next course.

**Pull request merged**